src/utils/Sorter.js

```diff
@@ -1013,9 +1042,36 @@ export default Backbone.View.extend({
     if (this.moved && target) {
       const toMove = this.toMove;
       const toMoveArr = isArray(toMove) ? toMove : toMove ? [toMove] : [src];
-      toMoveArr.forEach(model => {
-        moved.push(this.move(target, model, lastPos));
-      });
+      let domPositionOffset = 0;
+      toMoveArr
```

Comment:
There are cases for the sorter where toMoveArr is `[undefined]`, which allows the drop from blocks, native D&D and sort of layers in Style Manager (you can see instead how these are not working properly). This is why I was suggesting to use something `toMoveArr.length > 1` or any other check to prevent sorting 😅